tests/utils.py

```diff
@@ -205,11 +207,35 @@ async def new_consumer(c, group, fmt="avro", trail=""):
     return resp.json()["instance_id"]
 
 
-def new_random_name(prefix):
-    suffix = hash(random.random())
+def new_random_name(prefix: str) -> str:
+    suffix = str(uuid.uuid4())[:8]
     return f"{prefix}{suffix}"
 
 
+def create_subject_name_factory(prefix: str) -> str:
+    return create_id_factory(f"subject-{prefix}")
+
+
+def create_field_name_factory(prefix: str) -> str:
+    return create_id_factory(f"field-{prefix}")
+
+
+def create_id_factory(prefix: str) -> str:
```

Comment:
This type is not correct, this function creates a factory, the factory is a function:
```suggestion
def create_id_factory(prefix: str) -> Callable[[], str]:
```

Reply:
Fixing!